packages/xrpl/webpack.config.js

```diff
@@ -55,6 +55,7 @@ function getDefaultConfiguration() {
     },
     resolve: {
       alias: {
+        crypto: false,
```

Comment:
TODO: do we want this ? @ckniffen

Reply:
https://github.com/XRPLF/xrpl.js/blob/6bdcf629364061a683b148f042547be19e47473f/packages/xrpl/webpack.config.js#L65C1-L69C9

Your branch still has crypto before this PR

Reply:
This can be removed after making this change https://github.com/XRPLF/xrpl.js/pull/2273#discussion_r1289345265.

Also make sure to update `test/webpack.config.js`